### ISSUE TITLE
GTKUI: "Adding files..." dialog needs progress bar...

### ISSUE THREAD

**Issue by mdooligan** [Alexey-Yakovenko/deadbeef#1552]
Not urgent by any means, just something to go in the TODO list.

With a cursory look at progress.c, it looks like it's been stubbed in, just not implemented quite yet.

**Comment by Alexey-Yakovenko** [comment]
It's not quite the case. If you see foobar2000 progress dialog, you would notice that it doesn't have the progress bar either.

This is because when you add e.g. 50000 files from a folder recursively, to calculate the total number of files -- you'd have to walk entire file tree, and detect all supported file types, BEFORE you start showing the progress bar.

But this process itself unfortunately takes about 99% of time.

TL;DR to show the progress bar, calculation needs to take place, which takes 99% of time of file add process, which makes the progress bar useless.
